pipeline/transform.py

```diff
@@ -51,10 +53,31 @@ def get_summary(prompt: str, transcript: str) -> ChatCompletion:
         ],
         temperature=0.1,  # low temperature to ensure the model doesn't diverge from the prompt
     )
-    print(completion.usage)
+    usage = completion.usage
+    completion_tokens = getattr(usage, "completion_tokens", 0)
+    prompt_tokens = getattr(usage, "prompt_tokens", 0)
+    total_tokens = getattr(usage, "total_tokens", 0)
+    cost = [completion_tokens, prompt_tokens, total_tokens]
+    logger.info("GPT-4o-mini usage cost: %s", cost)
     return completion
 
 
+def is_valid_participant(data: dict) -> bool:
```

Comment:
Maybe the data parameter could have a better variable name?

Reply:
Renamed to participant_dict. Thanks Miles.